kit/proto/generate.go

```diff
@@ -0,0 +1,2 @@
+//go:generate protoc --go_out=paths=source_relative:. --go-grpc_out=paths=source_relative:. score.proto
```

Comment:
The `kit/proto` directory is old code and should be deleted. It is not in master. Don't understand how this has reappeared.